.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,71 @@
+name: 🐞 Report a bug
+description: Create a bug report about unexpected behavior.
+labels: [kind/bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+        Please only use this form to submit problems with builds or the runtime of Unikraft unikernels.  This includes unexpected or undefined behavior, stacktraces, critical messages, and things just breaking or exploding.
+        
+        **Ensure you have tested against the latest changes on `staging` branch before submitting this form.**
+
+        For questions, concerns or help with running Unikraft, please check out our additional resources:
+
+        * [Unikraft Documentation](https://docs.unikraft.org)
+        * [Getting Started Guide](https://unikraft.org/getting-started)
+        * [Unikraft Discussions Forum](https://github.com/unikraft/unikraft/discussions)
```

Comment:
Should we still use the forum? Or should we ditch it in favor of Discord?

Reply:
We should add Discord too and keep for the forum as a public Q&A, FAQ and for RFCs.